### DISCUSSION TITLE
Latest version for MacOS

### DISCUSSION THREAD

**Discussion by kristofrk** [weaveworks/eksctl#3469] [Q&A]
I am trying to download the latest version of eksctl for Mac. I was hoping to pick up 0.40 to be able to use eksctl for Fargate, but I only get 0.35. Is that the latest? I am using brew.

**Comment by aclevername** [comment]
looks like the latest version of eksctl is in brew https://formulae.brew.sh/formula/eksctl

**Reply by kristofrk** [reply]
Thanks. I downloaded it directly in the meantime. :-)
